Makefile

```diff
@@ -99,7 +99,7 @@ test: manifests generate fmt vet envtest ## Run tests.
 ##@ Build
 
 build: generate fmt vet ## Build manager binary.
-	go build -o bin/manager -ldflags "-X github.com/redhat-openshift-ecosystem/operator-certification-operator/version.commit=$(SHAVERSION) -X github.com/redhat-openshift-ecosystem/operator-certification-operator/version.version=$(RELEASE_TAG)" main.go
```

Comment:
Why remove the SHAVERSION? I think that's still very useful, to know where a binary was built from. Not useful on the images, but definitely useful on the binary IMHO

Reply:
Calculating SHAVERSION requires copying .git as well into the builder. Should we?

Reply:
added.